src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.FruitOperation;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<FruitOperation> SHOP_STORE = new ArrayList<>();
```

Comment:
According to the encapsulation principle, it's a bad practice to make your class fields public. Consider changing the access modifier of `SHOP_STORE` to private and providing a public method to access it if necessary.